### ISSUE TITLE
pythonscript.gdnlib missing from release packages

### ISSUE THREAD

**Issue by paolobb4** [touilleMan/godot-python#59]
Hi, this is a great project you've started here and I'm very exited about it but I'd like to mention that the packages found in the [releases](https://github.com/touilleMan/godot-python/releases) page only contain the folder _pythonscript_ but are missing the file _pythonscript.gdnlib_ that's required by gdnative wich means you can't even check out godot-python withougt having to go though compiling from src wich could be a turnoff for many people either for lack of time or because it can be intimidating for newer programmers (I know it still is a little intimidating for me).
I get it's best to build it yourself and that's what I ended up doing but still this is an easy thing to remedy, and it's such a great tool that I would hate to see people shy away from it for such a silly thing. Maybe it should also be mentioned in the doccumentation that both **pythonscript/** and **pythonscript.gdnlib** are required to be within the godot project.

I hope I don't come out as too pretentious, just trying to add my two cents.

PS: just copy-pasting _pythonscript.gdnlib_ from the pong example dind't work for me, it would actually crush godot. Maybe because the releases are behind in respect to src?

**Comment by AndersonFirmino** [comment]
@paolobb4 I totally agree with you. I tried to use but did not understand the readme.md right. I wish it were explained more, I even managed to compile godot with this package but I could not make it work or use it at all. And neither does the godot option for PythonScript appear.

**Comment by paolobb4** [comment]
Yeah I've been reading some of the src from this morning and it turns out that _readme.md_ is outdated by atleast 2 months (even though there were a couple of minor changes) as evidenced by the "Map of the code" listing various files under the folder **cffi_bindings/** wich was made obsolete and removed by [commit 52f93aacaef2eb982c1880b115a7d4bbdeaea76e](https://github.com/touilleMan/godot-python/commit/52f93aacaef2eb982c1880b115a7d4bbdeaea76e) wich is 2 moths old and I'm not sure what each listed file was replaced with.

But ignoring that (because lbh, it's hard to keep the doccumentation updated), the instructions for building from source `godot-python$ scons platform=x11-64 backend=cpython release` work for me. You should find a new folder within **godot-python/build/** containing the build (something like **x11-64-cpython/**) and then copy the contents within your godot project.
Although I couldn't manage to build the tests as indicated by the readme I made a small test project and it works.
Just something to look out for: currently optional arguments are not working so you'll have to explicitly state them as the expected default (just look up the function in godot's help).

**Comment by AndersonFirmino** [comment]
Did you compile with which release of godot 3? And when you open this compiled version with Python does it offer to edit PythonScript inside it? 

I'm curious because a few days ago. I broke my head for hours without much success. I even managed to compile Godot. But I could not use Python with it.

**Comment by paolobb4** [comment]
oh... Well I'm using godot 3.0.1-2. I didn't have to recompile godot to get godot-python working, I just package godot-python like it's indicated in the readme.md, created a new poject in godot and copied the built godot-python into the project's folder; next time I open the project the plug in is working (when choosing the language for a new script .py is now an option)

**Comment by AndersonFirmino** [comment]
That sounds very easy. I'll get the latest version of this repository and try to do exactly as you're saying.

**Comment by AndersonFirmino** [comment]
It worked. I learned how to make it work properly. Thank you very much. :snake:  :cake:

**Comment by paolobb4** [comment]
Great, man! Glad to have been of help :smile:
